pkg/apply/apply.go

```diff
@@ -876,14 +877,30 @@ func (t *TopicApplier) updatePlacementRunner(
 			roundLabel,
 		)
 
-		err := t.updatePartitionsIteration(
+		// print metrics during each iteration
+		applyMetricConfig := util.ApplyMetricConfig{
+			CurrRound: round,
+			TotalRounds: numRounds,
+			TopicName: t.topicName,
+			ClusterName: t.clusterConfig.Meta.Name,
+			ClusterEnvironment: t.clusterConfig.Meta.Environment,
+			ToRemove: t.config.BrokersToRemove,
+		}
+		metricStr, err := util.MetricConfigStr(applyMetricConfig)
+		if err != nil {
+			log.Errorf("Error: %+v", err)
+		}
+		go util.PrintMetrics(metricStr, stop)
```

Comment:
something like EmitMetrics would be a better name than PrintMetrics which specifically implies a print() function, which wouldn't be the case if using a generalized stats package.